Code enhancements and basic product section done

realyuvishere opened the pull request:
> Absolute path provisions
> README updated
> public/ directory updated with sass codes
> code beautified for easier understanding
> npm scripts updates
> custom css rules will have to be written through scss, which will be
compiled using the CLI or scout-app
> views/ directory contains another directory named partials/ which has
the scripts and meta tags in different files, which will be used in all
the other views
> go through the README.md for more details

Commit:
Code enhancements and basic product section done

> Absolute path provisions
> README updated
> public/ directory updated with sass codes
> code beautified for easier understanding
> npm scripts updates
> custom css rules will have to be written through scss, which will be
compiled using the CLI or scout-app
> views/ directory contains another directory named partials/ which has
the scripts and meta tags in different files, which will be used in all
the other views
> go through the README.md for more details

Commit:
Merge remote-tracking branch 'ALPHAVIO/main' into main

Conflicts resolved from the main branch

Pranav016 reviewed (commented):
* Best practices for an open-source project were not followed. Too many files added and updated in one commit and PR. Commits need to modular and commit messages should be clear. `Code enhancements` is not a good commit message, try to be specific. Complete one feature/ file and then make a commit.
* How did you run the application? Customer and home router were not acquired in the index.js file and jsonwebtoken package is missing from package.json.
* Make small commits and continuously make different PRs for different features, such as updated README should have been on a different PR. I'll explain on call why this important.
* **Never commit to master/ main branch** -> Golden rule
* Follow PR template to make it easier for the maintainer to review it.

Commit:
'Products' page linked

> /products route linked in views/home.ejs
> missing routes also fixed which were neglected due to merge conflicts
